Chap_API_Struct.tex

```diff
@@ -2198,6 +2281,8 @@ \subsection{Set Environment Variable}
 Similar to \code{setenv} from the C API, this allows the caller to set an environment variable in the specified \code{env} array, which could then be passed to the \refstruct{pmix_app_t} structure or any other destination.
 
 \adviceuserstart
+This macro is \emph{not} to be used to modify or set environmental variables in the user's environment (i.e., the \code{environ} array). Changes to that array should be perfomed using the typical \code{setenv} or \code{putenv} functions.
```

Comment:
Per 1Q 2022 discussion. This sentence should be removed.

Reply:
Done and updated